src/Function.cpp

```diff
@@ -765,38 +722,76 @@ const std::string &Function::name() const {
     return contents->name;
 }
 
-const std::vector<std::string> &Function::args() const {
-    return contents->args;
+Definition &Function::definition() {
+    return contents->init_def;
+}
+
+const Definition &Function::definition() const {
+    return contents->init_def;
+}
+
+const std::vector<std::string> Function::args() const {
```

Comment:
Hmm, I think I see now why none of the results of Func::args() were not const references... :)